docs/developer-docs/latest/guides/slug.md

```diff
@@ -89,6 +89,42 @@ module.exports = {
 };
 ```
 
+If you want to write automated slug for child components as well, use:
```

Comment:
```suggestion
To write automated slug for child components as well, use the following:
```

Change requested to be consistent with our styleguide and rules for technical writing (please see [our handbook](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing) for more details) 🙂